src/main/java/net/sf/jsqlparser/expression/RowTypeConstructor.java

```diff
@@ -14,14 +14,11 @@
 import net.sf.jsqlparser.parser.ASTNodeAccessImpl;
 import net.sf.jsqlparser.statement.create.table.ColumnDefinition;
 
-public class RowConstructor extends ASTNodeAccessImpl implements Expression {
+public class RowTypeConstructor extends ASTNodeAccessImpl implements Expression {
```

Comment:
Why did you rename an existing class? This will break code already running on JSqlParser.

Reply:
I second that, please lets not rename Classes without a very strong reason and without any deprecation approach.
It would break code, examples and documentation -- for a rather exotic new Expression.

Reply:
```sql
select 'yes' from dual where (sysdate-5,sysdate) overlaps (sysdate-2,sysdate-1)
```

Based on the example above, I do see:
```
"(" <EXPRESSION> "," <EXPRESSION> ")" <OVERLAPS> "(" <EXPRESSION> "," <EXPRESSION> ")"
```

Maybe a:
```
<EXPRESSION_LIST> <OVERLAPS> <EXPRESSION_LIST>
```

Please lets not tamper with `RowConstructor` here, it is not a SQL Row at all. Completely different semantics.

Reply:
Thanks for the feedback, I did not really take into consideration the effect of the rename. I will change it!

Reply:
There is no such thing as an 'SQL Row' in the sql grammar? They explicitly have two 'types' of row-constructs, one that defines a row-type (used in casts etc), and one that defines values.